packages/button/.storybook/addons.js

```diff
@@ -1,5 +1,6 @@
-/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
-import addons from '@storybook/addons'
 import '@storybook/addon-actions/register'
```

Comment:
most of this is just reorg to match the rest of the packages. i called out the actual changes